<launch>
  <node name="joystick_node" pkg="joy" exec="joy_node"/>
  <node name="joystick_control_node" pkg="ateam_joystick_control" exec="joystick_control_node">
    <param name="command_topic_template" value="/sim_radio_bridge/robot_motion_commands/robot{}" />
    <remap from="~/joy" to="/joy"/>
  </node>
</launch>
